src/api/tasks/Tasks.controller.js

```diff
@@ -0,0 +1,372 @@
+"use strict";
+
+const { logEmitter, INFO, ERROR } = require("../../services/logging.service");
```

Comment:
I think the main thing we're missing this is the statusemitter calls to update our dashboard to show whats passed/failed - I think we might have to leave for now since the failures in this module are less likely e.g. council not found

Reply:
Might be worth a chat pal